middleware/auth.ts

```diff
@@ -0,0 +1,21 @@
+const isAuthenticated = async (): Promise<boolean> => {
+  const sessionToken = sessionStorage.getItem("AUTH_KEY");
```

Comment:
please create a file with keys, ex. `keys.ts` :
```
const SESSION_STORAGE_AUTH_KEY = "AUTH_KEY"
```
and:
```
const sessionToken = sessionStorage.getItem(SESSION_STORAGE_AUTH_KEY);
```
thanks to this, you can also use this variable in other places and we will be able to easily use it in other places or modify it

Reply:
done